v2/our-platform/api-reference/backtest-management.rst

```diff
@@ -0,0 +1,436 @@
+===================
+Backtest Management
+===================
+
+The QuantConnect.com API exposes methods for creating, reading, listing, updating, and deleting backtests in the user's account.
+
+|
+
+----------------------------------------------------------------
+
+Backtest management API calls return either a Backtest object, a Backtest List object, or a Backtest Report object.
+
+|
+
+Backtest Object
+---------------
+
+A Backtest objects describes a Backtest response packet from the QuantConnect.com API.
+
+Attributes
+==========
+
+.. list-table::
+   :header-rows: 1
+
+   * - Attribute
+     - Type
+     - Description
+   * - name
+     - string
+     - Name of the backtest.
+   * - note
+     - string
+     - Note on the backtest, attached by the user.
+   * - backtestId
+     - string
+     - Assigned backtest Id.
+   * - completed
+     - boolean
+     - True when the backtest is completed.
+   * - progress
+     - decimal
+     - Progress of the backtest in percent 0-1.
+   * - result
+     - A BacktestResult object (see full API reference for its attributes)
+     - Result-specific items from the backtest packet.
+   * - error
+     - string
+     - Backtest error message.
+   * - stacktrace
+     - string
+     - Backtest error stacktrace.
+   * - created
+     - string
+     - Date and time the project was created.
+   * - success
+     - Boolean
+     - Indicate if the API request was successful.
+   * - errors
+     - Array of strings
+     - List of errors with the API call, if any.
+
+|
+
+----------------------------------------------------------------
+
+Backtest List Object
+--------------------
+
+A Backtest List object contains a list of Backtest objects for a project.
+
+Attributes
+==========
+
+.. list-table::
+   :header-rows: 1
+
+   * - Attribute
+     - Type
+     - Description
+   * - backtests
+     - Array of Backtest objects
+     - List of backtest for a project.
+   * - success
+     - Boolean
+     - Indicate if the API request was successful.
+   * - errors
+     - Array of strings
+     - List of errors with the API call, if any.
+
+|
+
+----------------------------------------------------------------
+
+Backtest Report Object
+----------------------
+
+A Backtest Report object contains the report of a backtest.
+
+Attributes
+==========
+
+.. list-table::
+   :header-rows: 1
+
+   * - Attribute
+     - Type
+     - Description
+   * - report
+     - string
+     - HTML data of the report, with embedded base64 images.
+   * - success
+     - Boolean
+     - Indicate if the API request was successful.
+   * - errors
+     - Array of strings
+     - List of errors with the API call, if any.
+
+|
+
+----------------------------------------------------------------
+
+Create a Backtest
+-----------------
+
+Create a new backtest.
+
+Path
+====
+
+``POST`` /backtests/create
+
+Request
+=======
+
+.. code-block::
+
+    {
+      "projectId": 000000001,
```

Comment:
For projectId's use 12345678 as it doesn't have leading 0s.

For compileIds use "" strings as its a hash not a number